src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.csproj

```diff
@@ -1,34 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <Description>SQLite implementation of the System.Data.Common provider model.
-
-Commonly Used Types:
-Microsoft.Data.Sqlite.SqliteCommand
-Microsoft.Data.Sqlite.SqliteConnection
-Microsoft.Data.Sqlite.SqliteConnectionStringBuilder
-Microsoft.Data.Sqlite.SqliteDataReader
-Microsoft.Data.Sqlite.SqliteException
-Microsoft.Data.Sqlite.SqliteFactory
-Microsoft.Data.Sqlite.SqliteParameter
-Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>SQLite;Data;ADO.NET</PackageTags>
-    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)$(MSBuildProjectName).ruleset</CodeAnalysisRuleSet>
-    <IncludeSymbols>true</IncludeSymbols>
+    <Description>Metapackage that includes Microsoft.Data.Sqlite.Core and SQLitePCLRaw.bundle_green.</Description>
```

Comment:
Can we keep the description the same? The fact that it is a metapackage is an implementation detail that 99% of people won't care about.